src/components/notifications/notifications.svc.coffee

```diff
@@ -0,0 +1,36 @@
+angular.module 'mnoUiElements'
+  .service 'Notifications', ($log, toastr, MnoeNotifications) ->
+
+    NOTIFICATION_TYPE_MAPPING = {
+      reminder: 'info',
+      due_date: 'warning',
+      status_change: 'info',
+    }
+
+    @init = () ->
+      $log.debug("Notifications are enabled")
+      MnoeNotifications.get().then(
+        (response)->
+          notifications = response.data.plain()
+          for notification in notifications
+            notification_type = notification.notification_type
+            method = NOTIFICATION_TYPE_MAPPING[notification_type]
+            message = notification.message.split("\n").join("</br>")
+            title = notification.title
+            onHidden = ->
+              params = {object_id: notification.object_id, object_type: notification.object_type, notification_type: notification_type}
+              MnoeNotifications.notified(params)
+            toastr[method](message,title, {
+              closeButton: true,
+              autoDismiss: false,
+              timeOut: null,
+              tapToDismiss: true,
+              extendedTimeOut: 1000000,
```

Comment:
Did you set both `timeOut ` and `extendedTimeOut` to `0`?

The toastr was not disappearing after hovering it when I tried. Try this code in your console:
```
toastr = angular.element(document.querySelector('html')).injector().get('toastr')
toastr.info('<input type="checkbox" checked> Success!', 'With HTML', {
  allowHtml: true, extendedTimeOut: 0, timeOut: 0, closeButton: true
});
```